src/pages/Browse/index.tsx

```diff
@@ -119,17 +110,15 @@ export class BrowseBase extends React.Component<Props> {
     }
   }
 
-  // When selecting a new file to view, we do not want to preserve the hash in
```

Comment:
Maybe this comment should be added back underneath `preserveHash: false`? It might be helpful if someone thinks about adding hash support in the future.